rules/S7180/java/metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "title": "\"@Cache*\" annotations should only be applied on concrete classes",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "spring"
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-7180",
+  "sqKey": "S7180",
+  "scope": "Main",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "LOW",
+      "RELIABILITY": "MEDIUM",
+      "SECURITY": "LOW"
+    },
+    "attribute": "INTENTIONAL"
```

Maybe you can use logical, as the code will never behave as intended

```suggestion
    "attribute": "LOGICAL"
```